notebooks/features/causal_inference/Effects of Outreach Efforts.ipynb

```diff
@@ -0,0 +1,288 @@
+    "* Interventions: information about which incentive was given to a customer.\n",
+    "* Outcome: the amount of product the customer bought in the year after the incentives were given.\n",
+    "\n",
+    "\n",
+    "| Feature Name    | Type | Details                                                                                                                                     |\n",
+    "|-----------------|------|---------------------------------------------------------------------------------------------------------------------------------------------|\n",
+    "| Global Flag     | W    | whether the customer has global offices                                                                                                     | \n",
+    "| Major Flag      | W    | whether the customer is a large consumer in their industry (as opposed to SMC - Small Medium Corporation - or SMB - Small Medium Business)  |\n",
+    "| SMC Flag        | W    | whether the customer is a Small Medium Corporation (SMC, as opposed to major and SMB)                                                       |\n",
+    "| Commercial Flag | W    | whether the customer's business is commercial (as opposed to public secor)                                                                  |\n",
+    "| IT Spend        | W    | $ spent on IT-related purchases                                                                                                             |\n",
+    "| Employee Count  | W    | number of employees                                                                                                                         |\n",
+    "| PC Count        | W    | number of PCs used by the customer                                                                                                          |                                                                                      |\n",
+    "| Discount        | T    | whether the customer was given a discount (binary)                                                                                          |\n",
+    "| Revenue         | Y    | $ Revenue from customer given by the amount of software purchased                                                                           |\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "jupyter": {
+     "outputs_hidden": false,
+     "source_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# Import the sample multi-attribution data\n",
+    "# \"https://mmlspark.blob.core.windows.net/publicwasb/multi_attribution_sample.csv\"\n",
+    "\n",
+    "# Azure storage access info\n",
+    "blob_account_name = \"mmlspark\"\n",
+    "blob_container_name = \"publicwasb\"\n",
+    "blob_relative_path = \"multi_attribution_sample.csv\"\n",
+    "blob_sas_token = \"\"\n",
```

Comment:
our blob is public so this isnt strictly needed, if you want to allow SAS access for users though maybe set this to None and make the SAS auth setting in an if statement

Reply:
last time I tried it without SAS token configuration and it failed, but now it works, thanks for the comment!